dub.sdl

```diff
@@ -22,7 +22,9 @@ configuration "static" {
 	targetType "executable"
 	versions "VibeDefaultMain"
 	lflags "-lz" "-lssl" "-lcrypto" "-ldl" platform="posix"
-	dflags "-static" platform="posix"
+    
+	// gcc on alpine has issues with the ld.gold linker, use bfd instead
```

Comment:
Would be better to pass this as DFLAGS from alpine as ld.gold does drastically speed-up the build time locally.

Reply:
Unfortunately ld.gold has some issues on musl, so either BFD or LLD has to be used.